pyxem/signals/crystallographic_map.py

```diff
@@ -32,18 +35,43 @@
 def load_mtex_map(filename):
     """
     Loads a crystallographic map saved by previously saved via .save_map()
+
+    Columns:
+    1 = phase id,
+    2-4 = Euler angles in the zxz convention (radians),
```

Comment:
Should this say/be degrees?

Reply:
It's probably just a documentation issue then, I couldn't find any conversions (out of degrees) when I was looking.